salt/data-pipeline/nifi.sls

```diff
@@ -81,6 +81,8 @@ nifi-init-file:
     file.managed:
         - name: /lib/systemd/system/nifi.service
         - source: salt://data-pipeline/config/lib-systemd-system-nifi.service
+        - watch_in:
```

Comment:
re: `watch` vs `listen`. I imagine if a service configuration file changes then the service should be restarted immediately rather than wait until end of highstate.